pages/404.js

```diff
@@ -0,0 +1,35 @@
+import Image from 'next/image';
+import styled from 'styled-components';
+import jackCable from '../public/jackCable.jpg';
+
+export default function PageNotFound() {
+  return (
+    <>
+      <ImageWrapper>
+        <Image
+          alt="background image jack cable"
+          src={jackCable}
+          layout="fill"
+          objectFit="cover"
+        />
+      </ImageWrapper>
```

Comment:
You have the exact same code in the EditBackgroundImage component. Instead of writing it again, you could  just import and call <EditBackgroundImage />, could you not?

Reply:
good point. thanks! :)